### ISSUE TITLE
Breakpoints should allow targeting of mobile

### ISSUE THREAD

**Issue by mlenser** [nordnet/nordnet-ui-kit#346]
We need a mixin to target mobile specifically in a good way. In robosave we use
```
onMobileOnly: `@media (max-width: ${theme.breakpoints.sm - 1}px)`,
onDesktop: `@media (min-width: ${theme.breakpoints.sm}px)`,
```
This could allow us to solve complex cases like https://github.com/nordnet/nordnet-ui-kit/blob/3cf590662cc989b193e9fb47464ccbc8c0956be5/src/components/td/td-styles.jsx#L72

**Comment by bstream** [comment]
What I usually do is something like this:
```js
ruleName: {
  backgroundColor: 'red', // on mobile
  [theme.mixins.media('sm')]: { // on tablet and up
    backgroundColor: 'green',
  },
  [theme.mixins.media('md')]: { // on desktop
    backgroundColor: 'yellow',
  },
}
```

**Comment by mlenser** [comment]
That way works for cases where you want something on multiple breakpoints.

The case above is where we want a style on a specific breakpoint. In that case it was `display`. To make that work with the current paradigm you'd have to define the standard display type for the other cases.

**Comment by bstream** [comment]
You can also use it like this today:
```js
hiddenOnMobile: {
  display: 'none',
  [theme.mixins.media('sm')]: {
    display: 'block', // reset it to standard value
  },
},
```

**Comment by bstream** [comment]
We have these defined in next:

```js
mixins: {
  ...nnTheme.mixins,
  onMobileOnly: `@media only screen and (max-width: ${nnTheme.breakpoints.sm - 1}px)`,
  onDesktop: `@media only screen and (min-width: ${nnTheme.breakpoints.sm}px)`,
  onTabletOnly: `@media only screen and (min-width: ${nnTheme.breakpoints.sm}px) and (max-width: ${nnTheme.breakpoints.md - 1}px)`,
  },
```

Maybe something like that?

**Comment by mlenser** [comment]
That `reset to standard value` is what I'm suggesting is probably not the best practice for 2 reasons:

1. Extra styles are best avoided.
1. The default of a modified component might not always be the default of the standard component. For example a component might want to modify the display property or padding property and if the base component modifies those in certain sizes then either the standard component's default might overwrite it due to specificity or the modified component may need to write the same media query code replacing the normal default with the desired property.

**Comment by mlenser** [comment]
Ya, we use similar ones one robosave.

The mobile first approach is the best approach when we want a property to apply on all breakpoints, though overwritten in larger breakpoints. That method should be encouraged.

Max width methods should be used sparingly.

**Comment by bstream** [comment]
Solved in #371
